gulp-core-build-typescript/src/TSLintTask.ts

```diff
@@ -189,4 +195,17 @@ export class TSLintTask extends GulpTask<ITSLintTaskConfig> {
       (this.taskConfig.useDefaultConfigAsBase ? this._defaultLintRules : {}),
       this.taskConfig.lintConfig || {});
   }
+
+  private _writeLintRules(lintRules: any): fs.WriteStream { // tslint:disable-line:no-any
+    const tslintFilePath: string = path.resolve(this.buildConfig.rootPath, this.buildConfig.tempFolder, 'tslint.json');
```

Comment:
Yes. There is a option - `tslint.configFile` to config the `tslint.config` file.

> https://marketplace.visualstudio.com/items?itemName=eg2.tslint